indexer/indexer_tests/categories_test.cpp

```diff
@@ -119,6 +119,32 @@ UNIT_TEST(CategoriesHolder_Smoke)
   }
 }
 
+UNIT_TEST(CategoriesHolder_ReadableName)
```

Comment:
Please, write a test that creates a fake categories.txt and checks all our tricky logic for default category name selection.